### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tesla809**
Added CONTRIBUTING.md with some simple guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+All code changes happen through pull requests. They are the best way to propose changes to the codebase.
+
+We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```
Comment (cds-amal) [inline CONTRIBUTING.md:27]: These aren't necessary. There are no code tests currently

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Report bugs using Github [issues](https://github.com/cds-amal/fzf-eip/issues).
+
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/cds-amal/fzf-eip/issues). It's super easy!
+
+### Bug reports format
+
+Write bug reports with detail, background, and sample code.
+
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report written by someone else. We also think it's not a bad model. Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), another app developer.
+
```
Comment (cds-amal) [inline CONTRIBUTING.md:51]: This is too complicated for this project

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People _love_ thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+
+We are borrowing these from [Facebook's Guidelines](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md)
+
+- 2 spaces for indentation rather than tabs
+
```
Comment (cds-amal) [inline CONTRIBUTING.md:90]: We should use tools to enforce this and not burden contributors

**Review by cds-amal — CHANGES_REQUESTED** [review]
Good start. We should whittle away what doesn't make sense for this project

File: CONTRIBUTING.md (hunk 1 above)
Comment (tesla809) [reply CONTRIBUTING.md:27]: Agreed, Removing and adding reference to familarize with Gitflow workflow

File: CONTRIBUTING.md (hunk 2 above)
Comment (tesla809) [reply CONTRIBUTING.md:51]: Sweet, will remove.

File: CONTRIBUTING.md (hunk 3 above)
Comment (tesla809) [reply CONTRIBUTING.md:90]: Adding "Tools to enforce this to be determined."

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -21,13 +21,10 @@ All code changes happen through pull requests. They are the best way to propose
...
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes.
-5. Make sure your code lints.
-6. Issue that pull request!
-7. Review notes from team and update PR accordingly
-8. Once PR has been approved, the branch is deleted.
+2. Issue that pull request!
+3. Review notes from team and update PR accordingly
+4. Once PR has been approved, the branch is deleted.
+5. For more info see the [Github Flow](https://guides.github.com/introduction/flow/index.html).
```
Comment (cds-amal) [inline CONTRIBUTING.md:27]: just a note, in markdown you can use `1.` for every item and the renderer will sort it out

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+Report bugs using Github [issues](https://github.com/cds-amal/fzf-eip/issues).
+
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/cds-amal/fzf-eip/issues). It's super easy!
+
+### Bug reports format
+
+Write bug reports with detail, background, and sample code.
+
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report written by someone else. We also think it's not a bad model. Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), another app developer.
```
Comment (cds-amal) [inline CONTRIBUTING.md:47]: This adds overhead to raising issues. We can use the summary text from below
```suggestion
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+### Summary of Great Bug Reports
+
```
Comment (cds-amal) [inline CONTRIBUTING.md:50]:
```suggestion
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+Great Bug Reports tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [A stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that _anyone_ with a base R setup can run to reproduce what I was seeing
```
Comment (cds-amal) [inline CONTRIBUTING.md:56]:
```suggestion
  - Give sample command/code if you can. 
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People _love_ thorough bug reports. I'm not even kidding.
```
Comment (cds-amal) [inline CONTRIBUTING.md:61]:
```suggestion
We _love_ thorough bug reports. I'm not even kidding.
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+## Use a Consistent Coding Style
+
+We are borrowing these from [Facebook's Guidelines](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md)
+
+- Tools to enforce consistent style to be determined.
+
```
Comment (cds-amal) [inline CONTRIBUTING.md:68]: This should be an issue instead of in contributing

**Review by cds-amal — CHANGES_REQUESTED** [review]

**Comment by tesla809** [comment]
Resolved!

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+  - Give sample command/code if you can.
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+We _love_ thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+
+We are borrowing these from [Facebook's Guidelines](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (tesla809) [reply CONTRIBUTING.md]: 👀

**Review by cds-amal — APPROVED** [review]

**Pull request merged**